test/server/lib/helpers/TestServer.ts

```diff
@@ -170,3 +196,111 @@ export class TestServer {
     }
   }
 }
+
+// FIXME: found that TestProxyServer exist, what should I do? :'(
```

Comment:
Self-Answering: I feel like the two classes serve different purposes:
 - `TestProxyServer` is meant to be a pure test proxy, especially for webhooks;
 - `TestServerProxy` is meant to be a reverse-proxy, to be placed in front of doc and home workers;

I rename the latter so it is more explicit.